src/views/organisations/issueDetails.html

```diff
@@ -75,7 +75,7 @@
       rows: entry.fields
     }) }}
 
-    {% if issueEntitiesCount > 1 %}
+    {% if pagination.items | length > 1 %}
```

Comment:
standardize this to how all conditionals for should render pagination are done for standardized testing later